packages/docker-node/src/helper.ts

```diff
@@ -0,0 +1,26 @@
+import fs from 'fs'
+import { join } from 'node:path'
+import { request } from 'undici'
+
+export const generateDevConfig = (genesisAccountArgs: string[] = []): string => {
+  const generateGenesisAccount = (args: string) => `
+[[genesis.issued_cells]]
+capacity = 20_000_000_000_00000000
+lock.code_hash = "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
+lock.args = "${args}"
+lock.hash_type = "type"\n`
+  const extraConfig = genesisAccountArgs.reduce((a, b) => a + generateGenesisAccount(b), '')
+  const config = fs.readFileSync(join(__dirname, '../ckb', 'dev-template.toml'), 'utf-8')
+
+  return config + extraConfig
+}
+
+export const CKBLatestBinVersion = async (): Promise<string | undefined> => {
+  const { body } = await request('https://api.github.com/repos/nervosnetwork/ckb/releases/latest', {
+    method: 'GET',
+    headers: { Accept: 'application/vnd.github+json', 'User-Agent': 'CKB-Js-Kuai-DApp' },
```

Comment:
What does the custom user agent do in this function

Reply:
I found it in the GitHub document, if I didn't add this, 403 error would be responded from GitHub api.

Reply:
That weird

```bash
> await fetch('https://api.github.com/repos/nervosnetwork/ckb/releases/latest').then(res => res.json()).then(res => res.tag_name)
# 'v0.110.0'
```

works in Node.js@18.16.0

Reply:
And if it's specified in github document, it's better to append the link to it as an annotation

Reply:
> That's weird
> 
> 
> 
> ```bash
> 
> > await fetch('https://api.github.com/repos/nervosnetwork/ckb/releases/latest').then(res => res.json()).then(res => res.tag_name)
> 
> # 'v0.110.0'
> 
> ```
> 
> 
> 
> works in Node.js@18.16.0

Too weird. I will have a try again rest of day and if it does not work either, I will add the url of document as the annotion.